### ISSUE TITLE
Multiple tag lines overwrite

### ISSUE THREAD

**Issue by mgwidmann** [cabbage-ex/gherkin#4]
```
Feature: Something

  @first_tag
  @second_tag
  Scenario: A scenario
    Given Something
```
This should result in the tags being `["first_tag", "second_tag"]`, yet it only contains `["second_tag"]`

**Comment by rawkode** [comment]
This was fixed by #5

**Comment by mgwidmann** [comment]
👍

**Comment by mgwidmann** [comment]
Published `v1.3.0`
